lib/application/pages/home/collections/collections_page.dart

```diff
@@ -91,3 +94,51 @@ class CollectionsContents extends HookWidget {
     return widget.withSymmetricalPadding(context, horizontal: Spacing.medium);
   }
 }
+
+/// [CollectionsPage] empty state, given the current [collectionsVM] state.
```

Comment:
```suggestion
```

Seems like an irrelevant doc for me, which is incorrectly referring to be related to [collectionsVM] - [_CollectionsContents] is.